x-pack/test/functional/apps/lens/thresholds.ts

```diff
@@ -50,7 +50,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await (
           await testSubjects.find('lnsXY_yThresholdLeftPanel > lns-dimensionTrigger')
         ).getVisibleText()
-      ).to.eql('Static value: 4992.44');
+      ).to.eql('Static value: 4220.02');
```

Comment:
Why is it changing value? It seems like the chart is single series, shouldn’t the value stay the same in this case?

Reply:
I thought at first there was some change in the dataset as also the heatmap suite was having a similar problem.
I've investigated the issue and found out the was a problem for the stack chart computation which is now fixed.
I'll push some additional unit tests for the function tomorrow so it will avoid regressions.